pkg/server/handler/optimizor/plan_replayer_test.go

```diff
@@ -112,6 +113,7 @@ func TestDumpPlanReplayerAPI(t *testing.T) {
 	defer server.Close()
 
 	filename, fileNameFromCapture := prepareData4PlanReplayer(t, client, dom)
+	defer os.RemoveAll(replayer.GetPlanReplayerDirName())
```

Comment:
Please remember to run ```make bazel_prepare```.

Reply:
@hawkingrei  I did it and pushed the fix on  [df088c1](https://github.com/pingcap/tidb/pull/49314/commits/df088c15cdc26b2a7f57a6d167e0d72800219c64)  (and setup pre-commit by using `.pre-commit-config.yaml`).